packages/catalog/src/variants/display-type-results/item-queue-button.tsx

```diff
@@ -40,7 +40,7 @@ const ItemQueueButton = ({
     [wasAddedToQueue, isLoading, item, onAddedToQueue]
   );
 
-  const btnClassNames = clsx([
+  const btnClassNames = twMerge([
```

Comment:
Think the `twMerge` is here what's causing an issue, more specifically L45. Looks like `twMerge` may not support objects as args this way: https://github.com/dcastil/tailwind-merge/discussions/137#discussioncomment-3481605